presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -336,15 +336,18 @@ the standard Java types can't be used as they aren't able to represent the input
 data.
 
 To define a method handle which can accept *any* types, use ``@BlockPosition``
-in conjunction with the ``@BlockIndex`` parameters. This works for both scalar
-and aggregation function implementations. An example signature is provided
-below. 
+in conjunction with the ``@BlockIndex`` parameters. Similar to the
+``@SqlNullable`` annotation, use the ``@NullablePosition`` annotation to denote
+that the function should be called when the block position is null.
+
+This works for both scalar and aggregation function implementations. An example
```

Comment:
```suggestion
This works for both scalar and aggregation function implementations. 
```
Suggest deleting the sentence "An example signature is provided below." as the other examples in this section aren't introduced. If I'm overlooking something meaningful here, just let me know and then ignore this suggestion.

Reply:
fixed